_plugins/jekyll-notranslate.rb

```diff
@@ -1,14 +1,9 @@
 module Jekyll
   module NoTranslate
     def no_translate(text, words)
-        for word in words do
-            word = word.downcase
-            text = text.gsub(word, "<span class=\"notranslate\">#{word}</span>")
-            word = word.capitalize
-            text = text.gsub(word, "<span class=\"notranslate\">#{word}</span>")
-            word = word.upcase
-            text = text.gsub(word, "<span class=\"notranslate\">#{word}</span>")
-        end
+        all_words = words + words.map { |word| word.downcase } + words.map { |word| word.capitalize } + words.map { |word| word.upcase }
```

Comment:
You don't need this given that the regexp is case insensitive..

Reply:
I did not realize